SU-SWS/stanford_events_importer
# 7.x-3.11

# READY FOR REVIEW

https://github.com/SU-SWS/stanford_events_importer/compare/7.x-3.10...tag-7.x-3.11

## Steps to Review
1. See comparison link above
2. Ensure that versions in `.info` files are set to `7.x-3.11`
3. Ensure that the CHANGELOG reflects the changes since `7.x-3.10` (already approved in #84)
4. Approve
